moderngl_window/loaders/program/separate.py

```diff
@@ -10,29 +12,48 @@
 class Loader(BaseLoader):
     kind = 'separate'
 
-    def load(self) -> moderngl.Program:
+    def load(self) -> Union[
+        moderngl.Program,
+        moderngl.ComputeShader,
+        program.ComputeProgram,

```

Comment:
Wraps moderngl.Program and exposes moderngl.ComputeShader through attribute compute_shader.

Reply:
Drop `program.ComputeProgram` completely. Returning `Program`, `ComputeProgram` or `ReloadableProgram` is more than enough for now,